### PR TITLE
Added the changes for RTS_v3.0.0

### PR DESCRIPTION
**Pull request by Pavan-Mudunuri**
# Description

- Added support for PowerMax Laurel.
- Added bug fix for SRDF module.
- Added support for connection timeout for all modules.

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
||

# Checklist:

- [x] I have performed a self-review of my own code to ensure there are no formatting, pep8, linting, or security issues
- [x] I have performed Ansible Sanity test using --docker default
- [x] I have verified that new and existing unit tests pass locally with my changes
- [x] I have not allowed coverage numbers to degenerate
- [ ] I have maintained at least 90% code coverage
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] Backward compatibility is not broken

# How Has This Been Tested?
Please describe the tests that you ran to verify your changes. Please also list any relevant details for your test configuration

- [x] Functional testing
- [x] Regression testing
- [x] Unit testing

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,2 +1,2 @@
 urllib3>=1.26.7
-PyU4V>=9.1.2.0
\ No newline at end of file
+PyU4V>=9.1.2.0
```
Comment (Bhavneet-Sharma) [inline requirements.txt:2]: As we have dropped the 91 support, then, can we make it 9.2.x.x?

File: plugins/modules/volume.py [hunk 2]
```diff
@@ -117,7 +117,7 @@
     user: "{{user}}"
     password: "{{password}}"
     serial_no: "{{serial_no}}"
-    new_name: "Test_GOLD_vol_Renamed"
+    new_name:  "Test_GOLD_vol_Renamed"
```
Comment (Bhavneet-Sharma) [inline plugins/modules/volume.py:120]: remove extra space before `"`

File: plugins/modules/volume.py [hunk 3]
```diff
@@ -104,7 +104,7 @@
     user: "{{user}}"
     password: "{{password}}"
     serial_no: "{{serial_no}}"
-    size: 3
+    size:  3
```
Comment (Bhavneet-Sharma) [inline plugins/modules/volume.py:107]: remove extra space before `"`

File: plugins/modules/storagegroup.py [hunk 4]
```diff
@@ -294,7 +294,7 @@
     sg_name: "ansible_test_sg"
     service_level: "Diamond"
     srp: "SRP_1"
-    compression: true
+    compression: True
```
Comment (Bhavneet-Sharma) [inline plugins/modules/storagegroup.py:297]: it should be **true**

File: plugins/modules/storagegroup.py [hunk 5]
```diff
@@ -267,8 +267,8 @@
     sg_name: "parent_sg"
     state: "present"
     child_storage_groups:
-      - "pie"
-      - "bar"
+    - "pie"
```
Comment (Bhavneet-Sharma) [inline plugins/modules/storagegroup.py:270]: restore the changes.

File: plugins/modules/storagegroup.py [hunk 6]
```diff
@@ -252,8 +252,8 @@
...
```
Comment (Bhavneet-Sharma) [inline plugins/modules/storagegroup.py:255]: revert the changes.

File: plugins/modules/storagegroup.py [hunk 7]
```diff
@@ -233,12 +233,12 @@
...
     sg_name: "foo"
     target_sg_name: "foo_sg"
-    force: true
+    force: True
     state: "present"
     volumes:
-      - vol_id: "00028"
-      - vol_id: "00018"
-      - vol_name: "ansible-vol"
+    - vol_id: "00028"
```
Comment (Bhavneet-Sharma) [inline plugins/modules/storagegroup.py:239]: revert the changes.

File: plugins/modules/storagegroup.py [hunk 8]
```diff
@@ -233,12 +233,12 @@
     serial_no: "{{serial_no}}"
     sg_name: "foo"
     target_sg_name: "foo_sg"
-    force: true
+    force: True
```
Comment (Bhavneet-Sharma) [inline plugins/modules/storagegroup.py:236]: **true**

File: plugins/modules/storagegroup.py [hunk 9]
```diff
@@ -156,9 +156,9 @@
     password: "{{password}}"
     serial_no: "{{serial_no}}"
     sg_name: "foo"
-    service_level: "Diamond"
+    service_level:  "Diamond"
     srp: "SRP_1"
-    compression: true
+    compression: True
```
Comment (Bhavneet-Sharma) [inline plugins/modules/storagegroup.py:161]: **true**
Applicable for all such instances.

File: plugins/modules/storagegroup.py [hunk 10]
```diff
@@ -156,9 +156,9 @@
     password: "{{password}}"
     serial_no: "{{serial_no}}"
     sg_name: "foo"
-    service_level: "Diamond"
+    service_level:  "Diamond"
```
Comment (Bhavneet-Sharma) [inline plugins/modules/storagegroup.py:159]: Remove extra space before `"`

Applicable for all such instances.

File: plugins/modules/storagegroup.py [hunk 11]
```diff
@@ -183,9 +183,9 @@
     sg_name: "foo"
     state: "present"
     volumes:
-      - vol_id: "00028"
-      - vol_id: "00018"
-      - vol_id: "00025"
+    - vol_id: "00028"
```
Comment (Bhavneet-Sharma) [inline plugins/modules/storagegroup.py:186]: Revert the changes. 
Applicable for all such instances.

File: plugins/modules/srdf.py [hunk 12]
```diff
@@ -148,7 +148,7 @@
     remote_serial_no: "{{remote_serial_no}}"
     state: 'present'
     srdf_mode: 'Synchronous'
-    wait_for_completion: true
+    wait_for_completion: True
```
Comment (Bhavneet-Sharma) [inline plugins/modules/srdf.py:151]: bool values should be in small letters.
Fix for all occurrences.

File: plugins/modules/rdfgroup.py [hunk 13]
```diff
@@ -55,13 +55,14 @@
 - name: Get specific volume details of an RDF Group
   dellemc.powermax.rdfgroup:
     unispherehost: "{{unispherehost}}"
-    serial_no: "{{serial_no}}"
+    serial_no:  "{{serial_no}}"
```
Comment (Bhavneet-Sharma) [inline plugins/modules/rdfgroup.py:58]: remove extra space before `"`

File: plugins/modules/portgroup.py [hunk 14]
```diff
@@ -106,10 +106,10 @@
     portgroup_name: "{{portgroup_name}}"
     state: "present"
     ports:
-      - director_id: "FA-1D"
-        port_id: "5"
-      - director_id: "FA-2D"
-        port_id: "5"
+    - director_id: "FA-1D"
```
Comment (Bhavneet-Sharma) [inline plugins/modules/portgroup.py:109]: Fix indentation to old code

File: requirements.txt (hunk 1 above)
Comment (trisha-dell) [reply requirements.txt:2]: still supporting 9.1

**Review by Bhavneet-Sharma — APPROVED** [review]

**Review by meenakshidembi691 — APPROVED** [review]

**Comment by Bhavneet-Sharma** [comment]
Delete tests/sanity/ignore-2.13.txt file as well.

**Comment by Bhavneet-Sharma** [comment]
> > Delete tests/sanity/ignore-2.13.txt file as well.
> 
> 2.13 is renamed to 2.16

Yeah I see its placed. Thank You Pavan

**Review by Jennifer-John — APPROVED** [review]
